MIGRATION.md

```diff
@@ -0,0 +1,114 @@
+# Migrating from `dp-frontend-renderer` to `dp-renderer`
+
+## Migrating assets and models
+
+The frontend service is responsible for setting up the assets binary and source path.
+
+You should also store all assets and models within the application itself with the following structure:
+
+```md
+.
+├── asset                    # relevant templates & localisations from dp-frontend-renderer
+│   ├── templates          
+│   ├── localisations  
+|   |   ├──  service.en.toml
+|   └── └──  service.cy.toml
+└── model                    # relevant models from dp-frontend-models
+```
+
+## Updating the Makefile
+
+In order for `dp-renderer` to work correctly once the assets have been migrated over, we use `go-bindata` to generate a combined assets source file.
+
+Update the frontend app's `Makefile` with the following new commands so that `go-bindata` will generate this file:
```

Comment:
We have previously used the term frontend service on line 5, to avoid confusion I think it would be good to keep to the same term, either frontend app or frontend service.